patches/chrome-browser-ui-views-frame-browser_view.cc.patch

```diff
@@ -10,3 +10,11 @@ index 9a5cac0d9e07513c67a6d641104bf0f1ad67eec7..7f41fe737018f077fe17a5120ccc514f
    views::View::ConvertPointToTarget(src, dst, point);
    return dst->HitTestPoint(*point);
  }
+@@ -715,6 +716,7 @@ class BrowserView::ExclusiveAccessContextImpl
+   void UpdateExclusiveAccessBubble(
+       const ExclusiveAccessBubbleParams& params,
+       ExclusiveAccessBubbleHideCallback first_hide_callback) override {
++    EXCLUSIVE_ACCESS_CONTEXT_IMPL_UPDATE_EXCLUSIVE_ACCESS_BUBBLE
```

Comment:
TODO - just leaving a tracking comment here so I don't lose this when it gets rebased

Reply:
this looks like it could be replaced by an override on `UpdateExclusiveAccessBubble`?